spec/services/bulk_upload/sales/year2022/row_parser_spec.rb

```diff
@@ -521,4 +521,24 @@
       end
     end
   end
+
+  describe "inferences" do
+    context "when buyer not interviewed" do
+      let(:attributes) { valid_attributes.merge(field_6: "1", field_24: nil, field_30: nil, field_31: nil, field_32: nil, field_34: nil, field_36: nil, field_37: nil, field_39: nil, field_48: nil, field_49: nil) }
+
+      it "infers correctly" do
+        log = parser.log
+        expect(log["ecstat1"]).to eq(0)
```

Comment:
Do we also want to check that `noint` is correctly set as 2?

Reply:
have changed the line to do with that so now noint is literally just field_6, but sure why not :)